src/main/java/core/basesyntax/Machines/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Machines;
+
+public class Excavator extends Machine{
+
+  @Override
+  public void doWork() {
+    System.out.println("Excavator is excavating something.");
+  }
+
+  @Override
+  public void stopWork() {
+    System.out.println("Excavator has stopped excavating.");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It should indicate what specific action the `Excavator` has stopped doing.